src/coreclr/nativeaot/Runtime/Full/CMakeLists.txt

```diff
@@ -28,9 +28,11 @@ endif (CLR_CMAKE_TARGET_WIN32)
 
 add_library(Runtime.WorkstationGC STATIC ${COMMON_RUNTIME_SOURCES} ${FULL_RUNTIME_SOURCES} ${RUNTIME_SOURCES_ARCH_ASM} ${RUNTIME_ARCH_ASM_OBJECTS})
 add_dependencies(Runtime.WorkstationGC aot_eventing_headers)
+add_dependencies(Runtime.WorkstationGC eventpipe-enabled)
```

Comment:
Why do we need this dependency?

Reply:
> Why do we need this dependency?

To fix the error listed in https://github.com/dotnet/runtime/pull/100356/checks?check_run_id=26327770979

Reply:
This should be fixed by adding a dummy implementation to src\coreclr\nativeaot\Runtime\disabledeventtrace.cpp.

The tracing support is optional in native AOT. We build two static libraries `eventpipe-enabled` and `eventpipe-disabled`. Only one of these libraries should be linked in into the final binary, depending on whether the tracing support is on or off.

Reply:
Thanks @jkotas: it makes much more sense than what I did...
I updated the PR accordingly